AoE2ScenarioParser/objects/support/area_pattern/area_state.py

```diff
@@ -5,13 +5,12 @@
 
 class AreaState(IntEnum):
     """Enum to show the state of the Area object"""
-    FULL = 0
+    RECT = 0
     EDGE = 1
     GRID = 2
     LINES = 3
     CORNERS = 4
 
-    @staticmethod
-    def unchunkables() -> set[AreaState]:
-        """Returns the states that cannot be split into chunks"""
-        return {AreaState.FULL, AreaState.EDGE}
+    def is_chunkable(self: AreaState) -> bool:
+        """Return if the state is chunkable or not"""
+        return self not in {AreaState.RECT, AreaState.EDGE}
```

Comment:
This is incorrect. This should return the states that allow it to be split into chunks. Both of these **cannot** be split.